canonical-websites/www.ubuntu.com
# Ensure search icon is always consistent

## Done

Ensure search icon is always consistent by using same SVG is all instances, only changing color from white/black where appropriate

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about))
- Verify search icon is the same (apart from color) at all breakpoints

## Issue / Card

Fixes #1738

## Demo

http://www.ubuntu.com-1738-search-icon.demo.haus/about/
